src/blog/math-random.md

```diff
@@ -35,7 +35,7 @@ MWC1616 uses little memory and is pretty fast to compute, but unfortunately offe
 - The more significant upper half of the result is almost entirely dependent on the value of state0. The period length would be at most 232, but instead of few large permutation cycles, there are many short ones. With a badly chosen initial state, the cycle length could be less than 40 million.
```

Comment:
`232` should be `2<sup>32</sup>` (same on the line above, and for `252` as well)

Reply:
Same for `2n` which should be `2<sup>n</sup>`